Client/Frontend/Reader/ReaderModeHandlers.swift

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct ReaderModeHandlers {
+    static func register(webServer: WebServer) {
+        // Register our fonts, which we want to expose to web content that we present in the WebView
+        webServer.registerMainBundleResourcesOfType("ttf", module: "reader-mode/fonts")
+
+        // Register the handler that accepts /reader-mode/page?url=http://www.example.com requests
+        webServer.registerHandlerForMethod("GET", module: "reader-mode", resource: "page") { (request: GCDWebServerRequest!) -> GCDWebServerResponse! in
+            if let url = NSURL(string: request.query["url"] as? String ?? "<invalid>") {
```

Comment:
This `<invalid>` bit is kind of funky. Can you extract this string handling to an outer `if let` check so that we don't have to use this hack?